test/test/util/ui_utils.js

```diff
@@ -195,4 +195,18 @@ shaka.test.UiUtils = class {
 
     return canvas;
   }
+
+  /**
+   * Creates a IMA SDK script element for testing.
+   *
+   * @return {!HTMLScriptElement}
+   */
+  static createImaSdkScript() {
+    const script = /** @type {!HTMLScriptElement} */(document.createElement(
+        'script'));
+
+    script.src = 'https://imasdk.googleapis.com/js/sdkloader/ima3.js';
```

Comment:
I think we should just do this once, at test boot.  We shouldn't have individual suites adding or removing script tags, IMO.

Reply:
done!